### PR TITLE
Register ihsan404.is-a.dev

### PR DESCRIPTION
**Pull request by rwixy**
Added `ihsan404` via the registration website.

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
We don't support TXT verification.

**Comment by Ihsan-N** [comment]
changed to cname

**Comment by WilliamDavidHarrison** [comment]
> changed to cname

There have been no changes on the PR.

File: domains/ihsan404.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+{
+  "owner": {
+    "username": "Ihsan-N",
+    "email": "abuihsan341@gmail.com"
+  },
+  "record": {
+    "TXT": "replit-verify=4a08a93f-e142-4ae8-93f3-6cd7dc6304b7"
```
Comment (Ihsan-N) [inline domains/ihsan404.json:8]: 4a08a93f-e142-4ae8-93f3-6cd7dc6304b7.id.repl.co

**Review by rwixy — COMMENTED** [review]
4a08a93f-e142-4ae8-93f3-6cd7dc6304b7.id.repl.co

**Pull request closed without merging**